src/main/java/com/google/devtools/build/lib/remote/options/RemoteOptions.java

```diff
@@ -67,10 +56,15 @@
 
   @Option(
       name = "remote_cache",
+      oldName = "remote_http_cache",
       defaultValue = "null",
       documentationCategory = OptionDocumentationCategory.REMOTE,
       effectTags = {OptionEffectTag.UNKNOWN},
-      help = "HOST or HOST:PORT of a remote caching endpoint.")
+      help =
+        "A base URL of a caching service in the format PROTOCOL://HOST or PROTOCOL://HOST:PORT."
+            + "Supported protocols: http, https, grpc, grpcs. "
+            + "If no protocol provided, will default to gRPC."
+            + "Format: (http|https|grpc|grpcs)://host:port")
```

Comment:
Two things:
* Let's just say `URI` which implies the format. For example we also support basic auth `https://username:password@hostname.com:port/path`. 
* I don't see any implementation of `grpcs` in this change. I think you ll need to implement this in a separate change where you deprecate `--tls_enabled`.

How about this:
`help = "A URI of a caching endpoint. The supported schemas are http(s) and grpc. If no schema is provided we'll default to grpc. See https://docs.bazel.build/versions/master/remote-caching.html"`

Reply:
done, thanks!